mkdocs.yml

```diff
@@ -47,6 +47,7 @@ nav:
       - Cassandra: modules/cassandra.md
       - ChromaDB: modules/chromadb.md
       - Couchbase: modules/couchbase.md
+      - CockroachDB: modules/cockroachdb.md
```

Comment:
Let's keep this list alphabetical 🙂

Reply:
Haha - I could swear this was alphabetical 🙃

Reply:
Haha I had to double check cos you clearly made effort to keep it alphabetical 😆 Thanks for sorting it